### PR TITLE
[OrderPlacement API] make receiver field optional

### PR DESCRIPTION
**Pull request by fleupold**
Slight usability improvement for people using the API. The field is already nullable so it might as well be completely optional

### Test Plan
Before the following request:

> curl 'http://localhost:8080/api/v1/orders' \
                                                 -H 'accept: application/json' \
                                                 -H 'Content-Type: application/json' \
                                                 --data-raw '{
                                                 "sellToken": "0x6810e776880c02933d47db1b9fc05908e5386b96",
                                                 "buyToken": "0x6810e776880c02933d47db1b9fc05908e5386b96",
                                                 "sellAmount": "1234567890",
                                                 "buyAmount": "1234567890",
                                                 "validTo": 2617291477,
                                                 "appData": "0x0000000000000000000000000000000000000000000000000000000000000000",
                                                 "feeAmount": "12345678900000000",
                                                 "kind": "buy",
                                                 "partiallyFillable": true,
                                                 "signature": "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
                                                 "signingScheme": "eip712"
                                               }' --compressed

Would return

> Request body deserialize error: missing field `receiver` at line 13 column 1

With this change it complains about a the fee (which happens after parsing the order).

### CODE REVIEW

File: orderbook/openapi.yml [hunk 1]
```diff
@@ -349,6 +349,7 @@ components:
             owner (i.e. the order signer).
           $ref: "#/components/schemas/Address"
           nullable: true
+          required: false
```
Comment (bh2smith) [inline orderbook/openapi.yml:352]: Didn't we just learn that all parameters of an object are optional by default? We shouldn't have to make it explicitly not required....

Comment (bh2smith) [reply orderbook/openapi.yml:352]: If this is the case then I slightly prefer these semantics rather than having a list of all required fields below.

**Review by bh2smith — APPROVED** [review]

**Comment by bh2smith** [comment]
I believe this is related to #440.

File: orderbook/openapi.yml (hunk 1 above)
Comment (bh2smith) [reply orderbook/openapi.yml:352]: I think the error may be caused by the fact that `required` is meant to be at the end of the list and usually indented one less. At least according to the errors I am seeing in the build this is what I had on this failing build in a different PR

https://github.com/gnosis/gp-v2-services/runs/2235945020

**Review by vkgnosis — APPROVED** [review]

File: orderbook/openapi.yml (hunk 1 above)
Comment (fleupold) [reply orderbook/openapi.yml:359]: makes sense. Removed it...

**Pull request merged**